src/Compilers/CSharp/Test/Semantic/Semantics/UninitializedNonNullableFieldTests.cs

```diff
@@ -67,6 +67,25 @@ public void NoNonNullWarnings_CSharp7()
                 Diagnostic(ErrorCode.WRN_UnreferencedField, "G").WithArguments("C.G").WithLocation(4, 19));
         }
 
+        [Fact]
+        public void NoExplicitConstructors()
```

Comment:
What is this catching? That the uninitialized fields walker doesn't run for LangVersion.Latest?

Reply:
Good catch. Added `WithNonNullTypesTrue()`.

---
In reply to: [307936049](https://github.com/dotnet/roslyn/pull/37507#discussion_r307936049) [](ancestors = 307936049)